src/_locales/en/messages.json

```
@@ -179,6 +179,14 @@
         "message": "Custom API URL",
         "description": "Input label"
     },
+    "instanceDomain": {
+        "message": "Instance Domain",
+        "description": "Input label"
+    },
+    "instanceDomainPlaceholder": {
+        "message": "ex. iddqd.social",
```

```suggestion
        "message": "ex. stereophonic.space",
```

Should we reference one of the instances highlighted on the official site https://pleroma.social?

---

Listing `iddqd.social` is just a vanity thing as it's the instance I run. `stereophonic.space` has nothing to do with music. I recommend using my instance because, as it stands, it's the only one with a frontend that actually makes any use of the scrobble information submitted. Getting people to use the information submitted is a conversation I'm having on the other side of this conversation. @inverse

---

If you're still at all interested in this commit, I can send it to `shitposter.club`, which has this implemented as well, and it isn't mine, if you're worried about me ego wanking about it too much. @yayuyokitano @inverse